### ISSUE TITLE
vibed failed to build 

### ISSUE THREAD

**Issue by Shadowys** [rejectedsoftware/vibe.d#1765]
Screencast here:
http://asciinema.org/a/0s3yz04c27mhlf2bmuq1khf7o

System: 
```
Distributor ID:	elementary
Description:	elementary OS 0.4.1 Loki
Release:	0.4.1
Codename:	loki
```

dub version:
```
DUB version Version: 0.9.24-2, built on Apr 16 2016
```

dmd version:
```
DMD64 D Compiler v2.074.0
Copyright (c) 1999-2017 by Digital Mars written by Walter Bright
```

**Comment by ZombineDev** [comment]
The problem is that you have a seriously outdated version of dub. With dub 1.3.0 I get:
https://asciinema.org/a/5caisuo5lq721lh0w4fvy0yg0

**Comment by Shadowys** [comment]
I see. Updating dub manually fixed the problem. However, I got the outdated dub from the apt repo for debian systems.

**Comment by ZombineDev** [comment]
There two other easy ways of getting a D compiler & dub that I prefer over apt repositories:
1) Using the "Install Script": http://dlang.org/download.html
2) Using snapd: http://forum.dlang.org/post/yweilgshkzlonrpbvkkr@forum.dlang.org

I have been using 1) for quite a bit now and I find it most useful when I want to have several D compilers installed side by side (e.g. latest stable ldc, dmd nightly build and dmd beta) + it automatically downloads the latest version of dub.

**Comment by s-ludwig** [comment]
There is also a workaround on the 0.7.x branch since a while (8ca11ebf04383c33f059f5d6ff4bb9aeaea468ef), but it's not yet in a released version (will be in 0.7.32).

**Comment by Shadowys** [comment]
I see. Yeah I identified the issue being the comment where sdl failed to parse, but dub doesnt allow direct changes because it only allows fresh installs. However it being the older version I guess im going with the snapd solution.

Thanks for your time!
